# Copyright 2020 Tetrate
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

diff --git a/.bazelrc b/.bazelrc
index 3923fcd..f134813 100644
--- a/envoy/.bazelrc
+++ b/envoy/.bazelrc
@@ -4,7 +4,7 @@

 # Keep envoy.bazelrc up-to-date by run:
 # curl -sSL https://raw.githubusercontent.com/envoyproxy/envoy-wasm/master/.bazelrc > envoy.bazelrc
-import %workspace%/envoy.bazelrc
+import %workspace%/envoy/envoy.bazelrc

 # Overrides workspace_status_command
 build --workspace_status_command=tools/bazel_get_workspace_status
